### DISCUSSION TITLE
Allow right clicking on requests in the requests list

### DISCUSSION THREAD

**Discussion by himat** [Kong/insomnia#6545] [Ideas]
It's annoying that if I want to do something like rename a request or duplicate it etc, I need to click on the dropdown arrow specifically, which is small.

Instead, I keep expecting to be able to right click anywhere on the entire row to get the dropdown to show up. I think that would be a better UX.

![image](https://github.com/Kong/insomnia/assets/1156974/37c199d8-08ee-4109-b205-7c231e2f9790)

**Comment by icnahom** [comment]
I'm enjoying Insomia, except this simple drawback. It's so frustrating to see "Select All" pop up on a Right click.
![image](https://github.com/Kong/insomnia/assets/60504285/3d6e3035-69ee-478b-b0ab-60b3b6691c29)
